guides/security/acceptable_use_policy.md

```diff
@@ -0,0 +1,76 @@
+# Acceptable Use Policy
+
+This Acceptable Use Policy (AUP) details how all users of any Made Tech IT device or service must behave, and is based on Made Tech’s guiding principles of treating people fairly and like adults. We trust our team members to do the right thing, and this policy is here to help guide individuals when they seek advice in how they should conduct themselves in any situation.
+
+The AUP policy outlines the acceptable use of equipment and services at Made Tech and puts in place rules to protect everyone who uses them. Inappropriate use of equipment and services exposes both Made Tech and our clients to risks. These risks include virus attacks, compromise of network systems and services, and legal action. 
+
+As with any policy, there are consequences to non-compliance, including potential disciplinary action being taken which may lead to dismissal and/or criminal proceedings.
+
+Made Tech endeavours to continually improve all policies and if you have feedback, please contact the Operations Team.
+
+Below are the minimum expectations for accessing those services appropriate to your role and responsibilities.
+
+## I will act with integrity at all times
+
+I will use the Made Tech service for the purpose it is intended for at all times, and not do anything knowingly that could harm the integrity or reputation of Made Tech.
+
+## I will use any Made Tech issued devices properly
+I understand Made Tech issued devices are issued for work purposes and I agree to return them for inspection as and when required. I will return it on leaving the employment of Made Tech or when it is no longer required unless otherwise agreed with Made Tech prior to leaving.
+
+## Personal Use
+I will exercise good judgement at all times regarding the reasonableness of any personal use
+
+## I will enable  2-factor authentication where this is required
+2-factor authentication (2FA) requires two forms of authentication, often a password and a code, sometimes sent via SMS or an application. It helps protect user accounts in the case that your password is somehow exposed. 
+
+## I will ensure my laptop isn’t left unattended or unlocked
+Lock your screen or put your computer to sleep when you’re not around. Colleagues might not have the same permissions as you and visitors might be wandering around where you’re working. When not in use ‘at work’, you can store in the laptop lockers provided or ‘at home’, in a safe place where it can’t be seen. If you are using your device in a public place, don’t leave it unattended and be wary of those who may be watching over your shoulder. 
+
+## I will ensure I report the arrival of guests and visitors promptly
```

Comment:
I think this is covered elsewhere? Also feels strange as part of an AUP, which in my mind is more about how I use the devices/services provided for my work

Reply:
removed